yw_website/yw_website/settings.py

```diff
@@ -77,13 +81,19 @@
                 'django.template.context_processors.request',
                 'django.contrib.auth.context_processors.auth',
                 'django.contrib.messages.context_processors.messages',
+                'django.template.context_processors.request',
+
             ],
         },
     },
 ]
 
 WSGI_APPLICATION = 'yw_website.wsgi.application'
 
+AUTHENTICATION_BACKENDS = (
+    'django.contrib.auth.backends.ModelBackend',
+    'allauth.account.auth_backends.AuthenticationBackend',
```

Comment:
would you be willing to drop some links in the description of this merge request so we can all reference the authentication framework you used later?

Reply:
https://django-allauth.readthedocs.io/en/latest/